src/hubbleds/pages/01-spectra-&-velocity/__init__.py

```diff
@@ -185,7 +185,7 @@ def _update_seed_data_with_examples(example_data):
                 )[1]
 
             for component in data.main_components:
-                update[component.label].append(getattr(measurement, component.label, None))
+                update[component.label].append(getattr(measurement, component.label, None) or float('nan'))
```

Comment:
It may never be relevant here, but I think it makes sense to do something more like
```python
value = getattr(measurement, component.label, None)
if value is None:
    value = float('nan')
update[component.label].append(value)
```
If the value here is not `None` but is falsy (zero, empty string, etc.) the current line will change it to `nan`, which I suspect is not what we want.